Library/Homebrew/cmd/update.rb

```diff
@@ -41,8 +41,10 @@ def update
     master_updater.pull!
     master_updated = master_updater.updated?
     if master_updated
-      puts "Updated Homebrew from #{master_updater.initial_revision[0, 8]} " \
-           "to #{master_updater.current_revision[0, 8]}."
+      initial_revision_short = shortenRevision(master_updater.initial_revision)
+      current_revision_short = shortenRevision(master_updater.current_revision)
+      puts "Updated Homebrew from #{initial_revision_short} " \
+           "to #{current_revision_short}."
```

Comment:
I think it would be sufficient to call these `initial_short` and `current_short`, which would allow the `puts` to stay on a single line. I would usually also prefer more descriptive variable names, but those lines seem to be close enough to allow for something more succinct.